src/main/java/bsh/util/AWTConsole.java

```diff
@@ -350,4 +349,24 @@ public String toString() {
     // unused
     public void keyTyped(KeyEvent e) { }
     public void keyReleased(KeyEvent e) { }
+
```

Comment:
here I haven't taken out the streams yet. let's finalize Interpreter first, than we can come back to this one.